freqtrade/strategy/strategy_helper.py

```diff
@@ -50,10 +53,17 @@ def merge_informative_pair(dataframe: pd.DataFrame, informative: pd.DataFrame,
 
     # Rename columns to be unique
     date_merge = 'date_merge'
-    if append_timeframe:
+    if append_timeframe and not suffix:
```

Comment:
we'll clearly need this documented in the docstring - as well as on the doc page.

The code itself looks ok :+1:

Reply:
Sounds good, can do :+1: